README.md

```diff
@@ -210,18 +248,19 @@ Class | Method | HTTP request | Description
  - [DocumentPretranslating](docs/DocumentPretranslating.md)
  - [DocumentPretranslatingStatus](docs/DocumentPretranslatingStatus.md)
  - [DocumentQuote](docs/DocumentQuote.md)
- - [DocumentUpdateParameters](docs/DocumentUpdateParameters.md)
  - [DocumentWithSegments](docs/DocumentWithSegments.md)
  - [DocumentWithoutSegments](docs/DocumentWithoutSegments.md)
  - [DocumentWithoutSegmentsStatus](docs/DocumentWithoutSegmentsStatus.md)
+ - [DocumentWorkflow](docs/DocumentWorkflow.md)
  - [EditFilterMappingParameters](docs/EditFilterMappingParameters.md)
  - [Error](docs/Error.md)
  - [FileDeleteResponse](docs/FileDeleteResponse.md)
+ - [InlineObject](docs/InlineObject.md)
+ - [InlineObject1](docs/InlineObject1.md)
```

Comment:
do we expect there to be any InlineObjects here still? it probably doesn't impact how anything functions but it would make it more clear if these had a specific name

Reply:
hmmmm, let me check what in the swagger file is generating this, from documentation looks like it would be something that defines a parameter directly in the endpoint definition

Reply:
found the culprits, created this follow up pr for front https://github.com/lilt/front/pull/13196/files and re-generated the code with those changes on this pr, giving a name to those schemas